components/Hero/Hero.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from './hero.module.scss'
+import  Wave  from '../Wave/Wave'
+
+const Hero = () => {
+    return (
+      <div className={styles.hero__content_blue}>
+         <div className="content__dark py-4">
```

Comment:
.hero__content .content__style--dark

Reply:
is no possible use -- because I get error